Userland/Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -70,6 +73,7 @@ class LoadRequest {
     HashMap<String, String> m_headers;
     ByteBuffer m_body;
     Core::ElapsedTimer m_load_timer;
+    Optional<Page*> m_page;
```

Comment:
I have no idea if it'd work, but having an `optional nullable pointer` seems counter-productive, would it be possible to do something like `Optional<Page&>`?
```suggestion
    Optional<Page&> m_page;
```
This way, the code is explicit about having either "nothing" or a valid reference.